nautobot/dcim/tests/test_filters.py

```diff
@@ -758,16 +788,24 @@ def test_contact_email(self):
 
     def test_status(self):
         params = {"status": ["active", "planned"]}
```

Comment:
We should probably grab some statuses from the objects just in case these 2 happen to not be used in the random data